src/bidiMapper/domains/context/context.ts

```diff
@@ -86,9 +86,26 @@ export class Context {
 
   private _initializeEventListeners() {
     this._initializePageLifecycleEventListener();
+    this._initializeCdpEventListeners();
     this._handleBindingCalledEvent();
   }
 
+  private _initializeCdpEventListeners() {
+    this._cdpClient.on('event', async (method, params) => {
+      await this._eventManager.sendEvent(
```

Comment:
is the event is already handled by the bidi mapper will it be still sent as a CDP command?

Reply:
Yes, CDP events are always sent to client in case of client subscribed to them. It doesn't depend on if BiDi processed it or not.